### DISCUSSION TITLE
Multiple resolutions

### DISCUSSION THREAD

**Discussion by allanf181** [iptv-org/community#85] [Ideas]
### Describe your issue

The channel list doesn't include multiple resolutions, it would be nice to include it because of latency, where high resolutions don't load low resolutions would do.

**Comment by BellezaEmporium** [comment]
Best workaround would be to precise LD, MD or HD next to the name, if you do not have any adaptive playlist.

For the Korean channels, it would be Animax LD, Animax MD, Animax HD.

**Comment by allanf181** [comment]
I think the best thing would be to put all the streams, as much as repeated streams will appear, they will be from different servers, so each person can watch the stream with the best connection for them.

**Reply by BellezaEmporium** [reply]
> I think the best thing would be to put all the streams, as much as repeated streams will appear, they will be from different servers, so each person can watch the stream with the best connection for them.

You could also create custom m3u adaptive files that you could host somewhere, referencing all 3 qualities.

**Reply by allanf181** [reply]
An adaptive m3u works depending on the player used, but some players, like vlc, may try to raise the bitrate and won't go back if the stream stops.

**Comment by allanf181** [comment]
https://github.com/allanf181/adaptive-streams/blob/main/streams/kr/ChannelSeezn.m3u8

I created a repo with the files, soon I'll finish adding the files and open a pr changing the streams.

**Reply by BellezaEmporium** [reply]
Sure, i may create a script to create all of them at once, i'll make a PR on your profile once I have the time to make it

**Reply by allanf181** [reply]
https://allanf181.github.io/adaptive-streams/streams/kr/ChannelSeezn.m3u8
https://github.com/allanf181/adaptive-streams/commit/75e31abf1b30dde5010264e64400725ae9f072ff

**Reply by BellezaEmporium** [reply]
It seems you were way faster than me for this one. Nice job.

**Comment by allanf181** [comment]
https://github.com/iptv-org/iptv/pull/7845
